pom.xml

```diff
@@ -148,11 +148,11 @@
     <hadoop-three.version>3.2.4</hadoop-three.version>
     <hadoop.version>${hadoop-three.version}</hadoop.version>
     <slf4j.version>1.7.25</slf4j.version>
-    <log4j.version>1.2.17</log4j.version>
+    <reload4j.version>1.2.25</reload4j.version>
     <checkstyle.version>8.45.1</checkstyle.version>
     <maven.checkstyle.version>3.2.1</maven.checkstyle.version>
     <hbase.checkstyle.version>2.5.0</hbase.checkstyle.version>
-    <surefire.version>3.0.0-M5</surefire.version>
+    <surefire.version>3.0.0-M9</surefire.version>
```

Comment:
3.0.0 final has been released

Reply:
Updated to 3.0.0 and verified it's working fine.